src/infrastructure/input.cpp

```diff
@@ -102,3 +108,16 @@ mfem::Vector FromInlet<mfem::Vector>::operator()(const axom::inlet::Table& base)
   }
   return result;
 }
+
+serac::input::BoundaryConditionInputInfo FromInlet<serac::input::BoundaryConditionInputInfo>::operator()(
+    const axom::inlet::Table& base)
+{
+  serac::input::BoundaryConditionInputInfo result;
+  // Build a set with just the values of the map
+  auto bdr_attr_map = base["attrs"].get<std::unordered_map<int, int>>();
```

Comment:
I'm confused why this is an unordered map instead of a `std::vector`. Is it because lua stuff is implicitly maps instead of vectors?

Reply:
Yeah, everything in Lua is a table, and indices can be arbitrary, e.g.:
```lua
map = { [7] = "a", [13] = "c" }
```
which doesn't conform to the contiguity implied by `std::vector`.